### ISSUE TITLE
`handleResponseStatus': unexpected status code (401) received; while creating key 

### ISSUE THREAD

**Issue by kushbhandari** [phrase/phraseapp-ruby#28]
I am trying to use phraseapp-ruby gem to create keys in my phraseapp project but getting below error:

<pre>
lib/phraseapp-ruby/request_handler.rb:172:in `handleResponseStatus': unexpected status code (401) received; expected 201 (RuntimeError)
	from /Users/kbhandari/.rvm/gems/ruby-2.3.1/gems/phraseapp-ruby-1.2.4/lib/phraseapp-ruby/request_handler.rb:154:in `http_send'
	from /Users/kbhandari/.rvm/gems/ruby-2.3.1/gems/phraseapp-ruby-1.2.4/lib/phraseapp-ruby/request_handler.rb:112:in `send_request'
	from /Users/kbhandari/.rvm/gems/ruby-2.3.1/gems/phraseapp-ruby-1.2.4/lib/phraseapp-ruby.rb:2382:in `key_create'
</pre>

Below is the code snippet which tries to create key:

<pre>
credentials = PhraseApp::Auth::Credentials.new(token: "MY_TOKEN")
client = PhraseApp::Client.new(credentials)
rsp, err = client.projects_list(1, 10)

params = PhraseApp::RequestParams::TranslationKeyParams.new(:name => "txt.test.key", :description => "test description", :tags => "test, first")
rsp = client.key_create('MY_PROJECT_ID', params)
</pre>

I confirmed that my access token is valid as I am able to list all my projects in phraseapp. The project I am trying to create the key in is also valid. 
What could be the possible cause for this error ?

**Comment by sacry-dyn** [comment]
Hi @kushbhandari 

From your path it looks like that you are using 

phraseapp-ruby-1.2.4

we are currently on 

```
$ gem search phraseapp-ruby
*** REMOTE GEMS ***

phraseapp-ruby (1.3.3)
```

Could you try to update the gem and see if the problems persist?

`gem update phraseapp-ruby`

**Comment by kushbhandari** [comment]
Issue is still the same:

<pre>
gems/phraseapp-ruby-1.3.3/lib/phraseapp-ruby/request_handler.rb:163:in `handleResponseStatus': unexpected status code (401) received; expected 201 (RuntimeError)
</pre>

**Comment by sacry-dyn** [comment]
@kushbhandari Okay, thank you for clarifying. Would you write a mail to us https://phraseapp.com/contact or support@phraseapp.com to discuss this further. We need some more details. As this is related to authentication we should probably not publish any information on a public repository.

**Comment by kushbhandari** [comment]
@sacry-dyn thanks for the response, I figured that I didin't have enough permissions to create/update keys. Though I faced another issue: Is it possible to create translation key with default text (say for english language alone) ? `TranslationKeyParams` doesn't have any param to specify that. How can we achieve that or once key is created how can we attach a default english translation with that key ?

**Comment by sacry-dyn** [comment]
@kushbhandari hey sorry for letting you wait on this (sickness and holiday). I believe you might have figured this out yourself by now. Currently, you cannot do this in one go. You have to create the key and use your default locale id in combination with your newly created key id to set the content in a second request.

The official documentation to our API might help here as well:

https://phraseapp.com/docs/api/v2/keys/
https://phraseapp.com/docs/api/v2/translations/

Regards,
Matthias
